### DISCUSSION TITLE
discussions: Layout bug

### DISCUSSION THREAD

**Discussion by lemon-mint** [community/community#3264] [Discussions]
![bug](https://user-images.githubusercontent.com/55233766/106349263-1487c200-6310-11eb-921f-e87c16cca671.png)

[Original link](https://bit.ly/2NO3SH4)

Environment: Chrome 88.0.4324.104 windows

**Comment by ije** [comment]
has same issue:
https://github.com/alephjs/aleph.js/discussions/91

<img width="1530" alt="Screen Shot 2021-01-31 at 17 26 51" src="https://user-images.githubusercontent.com/2883484/106379818-8a139100-63e9-11eb-88db-1f0b7f530dc5.png">
